### ISSUE TITLE
Accessibility false reports regarding <dl>, <dd>, and <dt>

### ISSUE THREAD

**Issue by RoryDuncan** [GoogleChrome/lighthouse#5940]
Lighthouse's Accessibility sections is reporting — and marking down — the following html structure, claiming it fails the 'Elements are Well-Structured' test.

``` html
  <dl>
    <div>
      <dt>VIN dt</dt>
      <dd>VIN dd</dd>
    </div>
    <div>
      <dt>VIN dt</dt>
      <dd>VIN dd</dd>
    </div>
  </dl>
```

The above abides by [specification for the `<dl>` tag](https://www.w3.org/TR/html5/grouping-content.html#the-dl-element):

> The dl element represents a description list of zero or more term-description groups. Each term-description group consists of one or more terms (represented by dt elements) possibly as children of a div element child, and one or more descriptions (represented by dd elements possibly as children of a div element child)**, ignoring any nodes other than dt and dd element children, and dt and dd elements that are children of div element children within a single dl element.

#### Steps to reproduce
1. Run LH on https://s.codepen.io/RoryDuncan/debug/wEgEWR/NjkYzqWobpBM
2. Go to **Elements are Well-Structured** section of **Accessibility**.
3.  Observe warnings listed.

#### Resulting Report:
![image](https://user-images.githubusercontent.com/766828/44921927-c7473680-acf8-11e8-8d61-bc146c58a335.png)

#### Case that Screen Readers may produce confusing or inaccurate output

Both ChromeVox and Windows Narrator skip over <dl> elements completely. 🤷‍♂️

**Comment by RoryDuncan** [comment]
oh cool, thanks for the info, Justin. 😄 
I subscribed to that pull request — I'll close this when it has been merged.
